qmetaobject/src/qtdeclarative.rs

```diff
@@ -44,19 +44,29 @@ cpp! {{
         }
     };
 
+    struct Arguments {
+        int argc;
+        char **argv;
+
+        ~Arguments() {
+            for (int i=0; i<argc; ++i) {
+                delete[] argv[i];
```

Comment:
i don't think it is safe to use the C++ de-allocation function for something that was allocated from Rust.

Reply:
I guess passing the actual arguments from `std::env::args()` would be a better solution, yes. The QT_QPA_PLATFORM environment variable is one that I wansn't able to find somehow, that actually gave me a better solution to what I was trying to achieve, so I don't really need this change anymore. I could still follow through on it with the `std::env::args()` route though, if it's still wanted?